src/app/analyze/stack/stack-details/stack-details.component.scss

```diff
@@ -0,0 +1,34 @@
+//.render-stack-tiles {
+//  padding: 1em;
+//  background: #eeeeee;
+//  border: 2px solid #fff;
+//}
+
+.modal-container {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  height: 100%;
+  font-family: 'Segoe UI Light', Helvetica, Arial, sans-serif;
```

Comment:
It would be good if you put this font family in the variable SCSS file and use that variable here.